dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/SdkFeature.kt

```diff
@@ -85,6 +86,16 @@ internal abstract class SdkFeature<T : Any, C : Configuration.Feature> {
         return featurePlugins
     }
 
+    fun flushStoredData() {
+        getDataFlusher().flush()
+    }
+
+    fun getDataFlusher(): DataFlusher {
```

Comment:
Why do you need this method, why not create it directly in the `flushStoredData()` method.

Reply:
Just for testing purposes